src/components/AdminModal.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from "react";
+import { List } from "antd";
+import axios from "axios";
+import { User } from "../types/type";
+
+const AdminModal = () => {
+  const [userDetails, setUserDetails] = useState<User>();
+  const token = localStorage.getItem("token");
```

Comment:
Fetch the token from redux instead of localstorage. Accessing localstorage again again is a expensive operation